src/main/java/org/radarbase/fcm/upstream/XmppFcmReceiver.java

```diff
@@ -59,19 +59,21 @@
 
   // We need to use an Xmpp sender here as we need to send ACK messages back to FCM server which is
   // not possible with Admin SDK.
-  private final transient XmppFcmSender fcmSender = new XmppFcmSender();
+  private final transient XmppFcmSender fcmSender;
```

Comment:
@yatharthranjan The `fcmSender` here was coming out as `null`. Is this change fine?

Reply:
Hi, it was because of XmppFcmSender not being a spring Component and hence could not autowire its dependencies.
This has been fixed in https://github.com/RADAR-base/RADAR-Appserver/pull/91. I would suggest to review that first and merge it to dev so this branch can also merge the changes.

Reply:
ok, it is merged now. You can pull the changes into this branch.

Reply:
Okay I see. Thanks!